bench/wtperf/wtperf_run_py/wtperf_run.py

```diff
@@ -61,14 +61,6 @@ def create_test_stat_path(test_home_path: str):
     return os.path.join(test_home_path, test_stats_file)
 
 
-def find_stat(test_stat_path: str, pattern: str, position_of_value: int):
```

Comment:
This is dead code. The actual `find_stat` is in `wtperf_stat_collections.py`

Reply:
It looks like I didn't delete it when I refactored the code earlier. Thanks for removing it.